src/kernels/jupyter/launcher/liveshare/hostJupyterServer.node.ts

```diff
@@ -162,7 +170,8 @@ export class HostJupyterServer implements INotebookServer {
         resource: Resource,
         kernelConnection: KernelConnectionMetadata,
         cancelToken: CancellationToken,
-        ui: IDisplayOptions
+        ui: IDisplayOptions,
+        creator: KernelActionSource
```

Comment:
Could these all be defaulted so that it only needs to be set when using the 3rd party stuff?

Reply:
Yes they could (that's what I thought of), 
but i felt it was best to be explicit, else we could have scenarios where this isn't being setup correctly.
Will change as per the suggestion.

Reply:
Based on the current code path, we only need to pass a value in just 3 places (when starting a kernel for 3rd party, notebook & IW). Hence I think we don't need to default this.